packages/nc-gui/composables/useMultiSelect/index.ts

```
@@ -30,27 +30,35 @@ const MAIN_MOUSE_PRESSED = 0
  */
+
 export function useMultiSelect(
   _meta: MaybeRef<TableType | undefined>,
   fields: MaybeRef<ColumnType[]>,
-  data: MaybeRef<Row[]>,
+  data: MaybeRef<Row[]> | MaybeRef<Map<number, Row>>,
+  _totalRows?: MaybeRef<number>,
   _editEnabled: MaybeRef<boolean>,
   isPkAvail: MaybeRef<boolean | undefined>,
   contextMenu: Ref<boolean>,
   clearCell: Function,
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)